gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Mon May 21 14:01:33 ART 2018
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.0-all.zip
```

Comment:
You should upgrade the Gradle distribution to distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-all.zip

Currently doesn't work on my Mac with Java 9 or 10.